components/mpas-ocean/src/shared/mpas_ocn_tracer_ideal_age.F

```diff
@@ -112,30 +112,35 @@ subroutine ocn_tracer_ideal_age_compute(nTracers, nCellsSolve, minLevelCell, max
 
       integer :: iCell, iLevel, iTracer
 
-      !move to ocean constants
-      real (kind=RKIND), parameter :: c0 = 0.0_RKIND
-      real (kind=RKIND), parameter :: c1 = 1.0_RKIND
-
       err = 0
 
+      ! add a tendency increment equivalent to "dt" to entire domain below top level
       !$omp parallel
       !$omp do schedule(runtime) private(iLevel, iTracer)
       do iCell=1,nCellsSolve
-        do iLevel=minLevelCell(iCell),maxLevelCell(iCell)
+        do iLevel=minLevelCell(iCell) + 1, maxLevelCell(iCell)
```

Comment:
@maltrud, are you sure that this is correct?  Don't you want to increment even in the top layer because you are only resetting to zero in the open ocean, not under ice shelves?